ml_service/pipelines/diabetes_regression_build_train_pipeline.py

```diff
@@ -89,6 +89,7 @@ def main():
         arguments=[
             "--build_id", build_id_param,
             "--model_name", model_name_param,
+            "--allow_run_cancel", e.allow_run_cancel,
```

Comment:
This still isn't used here.  You need to match with **run_evaluation**

Reply:
YES it is :) evaluate_model.py line 87 and lone 107